src/routes/Project/components/ProjectParams.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+import './ProjectParams.scss';
+
+import ProjectConstants from '../constants';
+
+import common from 'common';
+
+export default class ProjectParams extends Component {
+  constructor(props) {
+    super(props);
+    this.handleTimeDivChange = this.handleTimeDivChange.bind(this);
+    this.handleTimeSigChange = this.handleTimeSigChange.bind(this);
+    this.handleTempoChange =   this.handleTempoChange.bind(this);
+  }
+
+  componentDidMount() {
+    ;
+  }
+
+  handleTempoChange(event) {
+    // not supported yet
```

Comment:
theres no support for actually doing anything with either thing yet, but they would be implemented in the same way as the other one.